.github/workflows/quickstart_ejb-txn-remote-call_ci.yml

```diff
@@ -20,7 +20,8 @@ env:
   MATRIX_JDK: '"11", "17"'
   MATRIX_OS: '"ubuntu-latest"'
   SERVER_PROVISIONING_SERVER_HOST: 'http://localhost'
-
+  EXTRA_RUN_ARGS: '-Pprovisioned-server -DpostgresqlUsername="test" -DpostgresqlPassword="test"'
```

Comment:
-Pprovisioned-server as a run arg looks weird, may come back after checking the QS changes